etc/adminhtml/system.xml

```diff
@@ -110,9 +110,30 @@
                             <field id="active">1</field>
                         </depends>
                     </field>
-                    <field id="pricing_strategy" showInDefault="1" showInStore="1" showInWebsite="1" sortOrder="100" translate="label" type="select">
+                    <field id="pricing_strategy" showInDefault="1" sortOrder="100" translate="label" type="select">
                         <label>DHL Pricing Strategy</label>
-                        <source_model>Reach\Payment\Model\Config\Source\Pricing</source_model>
+                        <source_model>Reach\Payment\Model\Config\Source\PricingStrategy</source_model>
+                        <depends>
+                            <field id="active">1</field>
+                        </depends>
+                    </field>
+                    <field id="clearance_mode" showInDefault="1" sortOrder="100" translate="label" type="select">
+                        <label>Clearance Mode</label>
+                        <source_model>Reach\Payment\Model\Config\Source\ClearanceMode</source_model>
```

Comment:
just to check, is the default value determined by the sort order? Or is there a way to explicitly set what the default value should be? (one of the requirements is that "Courier" is the default value for ClearanceMode for example)